build-parent/pom.xml

```diff
@@ -37,7 +37,6 @@
         <graal-sdk.version-for-documentation>20.3.1</graal-sdk.version-for-documentation>
         <mandrel.version-for-documentation>20.3</mandrel.version-for-documentation>
         <rest-assured.version>4.3.2</rest-assured.version>
-        <axle-client.version>1.1.0</axle-client.version>
         <mutiny-client.version>1.1.0</mutiny-client.version>
```

Comment:
This property doesn't seem to be referenced anymore? Or is there some black magic in the `docs` module?

Reply:
@cescoffier Maybe you want to have a look at this ^ even though the PR has been merged already.